src/metabase/pulse/render/png.clj

```diff
@@ -73,16 +74,65 @@
           (.getSubimage image 0 0 content-width (.getHeight image))
           image)))))
 
+(defn- font-can-fully-render?
+  [^Font font s]
+  (neg? (.canDisplayUpTo font s)))
+
+(def ^:private get-lato
```

Comment:
If you add `^{:arglists '(^java.awt.Font [])}` metadata on this then you can use `(get-lato)` directly in Java interop without having to worry about type hinting it

Reply:
Isn't it ok to put `def ^:private get-lato ^Font [...]` directly? `:arglists` are more for varargs thingie I guess and it could be simpler here?